components/molecule/field/src/index.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomValidationText, {
+  AtomValidationTextTypes
+} from '@s-ui/react-atom-validation-text'
+import AtomLabel, {AtomLabelTypes} from '@s-ui/react-atom-label'
+import AtomHelpText from '@s-ui/react-atom-help-text'
+
+const BASE_CLASS = 'sui-MoleculeField'
+
+class MoleculeField extends Component {
+  getClassNames(inline) {
+    return cx(BASE_CLASS, inline && `${BASE_CLASS}--inline`)
+  }
+
+  getTypeValidation(element) {
+    if (this.props.successText) {
+      if (element === 'label') return AtomLabelTypes.SUCCESS
+      if (element === 'validationText') return AtomValidationTextTypes.SUCCESS
+    }
+    if (this.props.errorText) {
+      if (element === 'label') return AtomLabelTypes.ERROR
+      if (element === 'validationText') return AtomValidationTextTypes.ERROR
+    }
+  }
+
+  get statusValidationText() {
+    if (this.props.successText) return this.props.successText
+    if (this.props.errorText) return this.props.errorText
```

Comment:
what if the developer passed through a success but also an error message? is it just a corner case that shouldn't happen? I intuitively think that the error message should be shown as it is more restrictive